.codeclimate.yml

```diff
@@ -0,0 +1,82 @@
+# This is a sample .codeclimate.yml configured for Engine analysis on Code
+# Climate Platform. For an overview of the Code Climate Platform, see here:
+# http://docs.codeclimate.com/article/300-the-codeclimate-platform
+
+# Under the engines key, you can configure which engines will analyze your repo.
+# Each key is an engine name. For each value, you need to specify enabled: true
+# to enable the engine as well as any other engines-specific configuration.
+
+# For more details, see here:
+# http://docs.codeclimate.com/article/289-configuring-your-repository-via-codeclimate-yml#platform
+
+# For a list of all available engines, see here:
+# http://docs.codeclimate.com/article/296-engines-available-engines
+
+#engines:
+# to turn on an engine, add it here and set enabled to `true`
+# to turn off an engine, set enabled to `false` or remove it
+#  rubocop:
+#    enabled: true
+#  golint:
+#    enabled: true
+#  gofmt:
+#    enabled: true
+#  eslint:
+#    enabled: true
+#  csslint:
```

Comment:
Can you please add SonarJava?

https://docs.codeclimate.com/v1.0/docs/sonar-java

Reply:
Hi , I have added SonarJava.